fake-lmp-device-register

```diff
@@ -85,8 +93,11 @@ def main(opts: Options):
 
     r = requests.post(opts.registration_url, json=data)
     if r.status_code != 201:
-        sys.exit("ERROR: HTTP_%d: %s" % (r.status_code, r.text))
-
+        log.error("ERROR: HTTP_%d: %s", r.status_code, r.text)
+        sys.exit(1)
+    else:
+        log.debug("Successfully registered device with status code: %d", r.status_code)
+        
```

Comment:
For some strange reason, one of my 2 comments got eaten by GitHub.
So, copying it again. Sorry for inconvenience.

```suggestion
    r = requests.post(opts.registration_url, json=data)
    r.raise_for_status()
    log.debug("Successfully registered device with status code: %d", r.status_code)

```

Rationate is the same: tracebacks are nice.

Reply:
I addressed your suggestion and I am getting this one merged